wg-manifests/charter.md

```diff
@@ -0,0 +1,100 @@
+# WG Manifests Charter
+
+This charter adheres to the conventions, roles and organization management
+outlined in [wg-governance].
+
+## Scope
+
+- Provide a catalog (centralized repository) of Kubeflow application manifests.
+- Provide a catalog of third-party apps for common services.
+- Provide documentation to help users install a set, or all of the included
+  apps.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Maintain a set of manifests that will allow users to install Kubeflow apps and
+  common services easily on Kubernetes, either on the cloud or on-prem, without
+  depending on external cloud services or closed source solutions. These
+  manifests will be developed with certain principles in mind, which are
+  outlined in the last bullet. Those include:
+    1. A common set of manifests for the current five (5) official Kubeflow
+       applications:
+        - Training Operators
+        - Kubeflow Pipelines (KFP)
+        - Notebooks
+        - KFServing
+        - Katib
+    1. Manifests for a set of specific common services:
+       - Istio
+       - KNative
+       - Dex
+       - Cert-Manager
+    1. Manifests for common, currently unmaintained, apps:
+        - Central Dashboard
+        - Profile Controller
+        - PodDefaults Controller
+- Maintain documentation and instructions for installing a set or all of the
```

Comment:
Though we should not rely on vendor distribution, I think we still need to deploy applications on a "generic" kubernetes system, what would this be?

Reply:
I think we will all have to decide this together as a WG, when we pin down how to test.